Cargo.toml

```diff
@@ -8,5 +8,6 @@ solana = "edge"
 # Specify Rust toolchains for rustfmt, clippy, and build.
 # Any unprovided toolchains default to stable.
 [workspace.metadata.toolchains]
-format = "1.79.0"
-lint = "1.79.0"
+build = "nightly"
+format = "nightly"
+lint = "nightly"
```

Comment:
Just for consistency with the other programs: `nightly-2023-10-05` (these might even be outdated by now).

https://github.com/solana-program/address-lookup-table/blob/7562c925f90459ad891d56719c6ea696ae9f555a/Cargo.toml#L11-L12

Reply:
That one is too old for the `v2.1` version.